Dockerfile

```diff
@@ -31,7 +31,7 @@ ADD nginx-$NGINX_VERSION.tar.gz /tmp/
 
 WORKDIR /tmp
 RUN wget https://sourceforge.net/projects/pcre/files/pcre/8.44/pcre-8.44.tar.gz && tar xzvf pcre-8.44.tar.gz
-RUN wget https://www.zlib.net/zlib-1.2.11.tar.gz && tar xzvf zlib-1.2.11.tar.gz
+RUN wget https://sourceforge.net/projects/libpng/files/zlib/1.2.11/zlib-1.2.11.tar.gz && tar xzvf zlib-1.2.11.tar.gz
```

Comment:
https://www.zlib.net/zlib-1.2.12.tar.gz works. Is it that the US site deletes old versions but the mirrors maintain them?

Should we update this to 1.2.12? I think that fixed a security issue in 1.2.11.